infrastructure/miner/Cargo.toml

```diff
@@ -20,7 +20,7 @@ tower = "=0.3.0-alpha.2"
 clap = "2.33.0"
 serde = { version = "1.0.101", features = ["derive"] }
 config = { version = "0.9.3" }
-env_logger = "0.7"
```

Comment:
and this one

Reply:
log4rs = { version = "0.8.3", features = ["toml_format"] }

you mean that one?

Reply:
Libraries just need the `log` crate; unless it was a dev-dependency